evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,373 @@
+                let width = active.outerWidth();
+                let x = event.originalEvent.touches[0].pageX - touchX;
+                let trigger = width / 10;
+
+                if(Math.abs(x) >= trigger) {
+                    resetTouchSliding();
+                    if(x > 0) {
+                        handle("slide-left");
+                    } else {
+                        handle("slide-right");
+                    }
+                    resetTouchSliding();
+                } else {
+                    active.css({
+                        transform: "translateX(" + x + "px)",
+                        opacity: 1 - Math.abs(x / trigger),
+                        transition: "none"
+                    });
+                    active.position();
+                    active.css("transition", "");
+                }
+            });
+            slider.on("touchend", resetTouchSliding);
+            slider.on("touchcancel", resetTouchSliding);
+
+            function resetTouchSliding() {
+                touchX = null;
+                findActive(2).css({transform: "", opacity: ""});
+            }
+
+            handle("repeat-unreviewed");
+        });
+
+        function handle(action, click) {
+            let parts = action.split("-");
+
+            if(parts[0] === "slide") {
+                slide(parts[1]);
+                updateButtons();
+            } else if(parts[0] === "repeat") {
+                repeat(parts[1]);
+                updateButtons();
+            } else if(action === "next-course" && !click) {
+                slider.find("[data-action=next-course]")[0].click();
+            } else if(["publish", "make_private", "hide", "unreview"].indexOf(action) !== -1) {
+                press(action, click);
+            }
+        }
+
+        function slide(to) {
+            let direction = to;
+            if(to === "left" && index > 0) {
+                index--;
+            } else if(to === "right" && index < items.length) {
+                index++;
+            } else if(Number.isInteger(to)) {
+                direction = to >= index? "right" : "left";
+                index = to;
+            }
+
+            if(index === items.length) {
+                let reviewed = items.every(item => item.is("[data-review]"));
+                let alert = slider.find(".alert");
+                alert.toggleClass("alert-warning", !reviewed);
+                alert.toggleClass("alert-success", reviewed && items.length !== 0);
+                alert.toggleClass("alert-secondary", reviewed && items.length === 0);
+                alert.find("[data-content=unreviewed]").toggleClass("d-none", reviewed);
+                alert.find("[data-content=reviewed]").toggleClass("d-none", !reviewed);
+                slider.find("[data-action=repeat-unreviewed]").toggleClass("d-none", reviewed);
+                slider.find("[data-action=repeat-all]").toggleClass("d-none", items.length === 0);
```

Comment:
I came up with the idea to have delegated classes `.reviewed` and  `.unreviewed` which would hide or show the content in css.